code/modules/mob/dead/observer/observer.dm

```diff
@@ -512,6 +512,9 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 
 	if(!target)
 		return 0
+	if(istype (target, /mob/living/simple_animal/hostile/megafauna))
```

Comment:
seems like it would be better to make a var, specifically, for this, similar to the gold core slime one or sentience potion, that way it's not as snowflaked and is "future proofed" if there's more than one of these mobs that come up.